department-of-veterans-affairs/caseflow
# Hold open diary updates

Resolves #6887 

To test:
1) Go to the daily docket
1) Change the hold open value
1) Refresh the page (or wait 30 seconds for the auto save to go through)
1) Confirm the note exists in VACOLS
1) Confirm the hold open value is updated correctly in VACOLS and the UI
